wiki/Achievements/it.md

```diff
@@ -6,7 +6,7 @@ Obiettivi
 Beatmap Packs
```

Comment:
alrighty, I was only focusing on image/links rather than contents for the most parts :/